MakieCore/src/basic_plots.jl

```diff
@@ -345,6 +351,8 @@ $(Base.Docs.doc(MakieCore.generic_plot_attributes!))
     linewidth = @inherit linewidth
     "Sets the pattern of the line e.g. `:solid`, `:dot`, `:dashdot`. For custom patterns look at `Linestyle(Number[...])`"
     linestyle = nothing
+    "Sets the type of linecap used, i.e. :butt (flat with no extrusion), :square (flat with 1 linewidth extrusion) or :round."
```

Comment:
Nitpicking, but is 1 linewidth extrusion correct? I think it's just half, because the line end is the center of the square, right?

Reply:
Yes that should only be half